src/sentry/static/sentry/app/components/pagination.tsx

```diff
@@ -32,7 +32,6 @@ type Props = {
 
 class Pagination extends React.Component<Props> {
   static propTypes = {
-    pageLinks: PropTypes.string,
```

Comment:
On line 28 we already have `pageLinks: string | null | undefined;`.
Removing it from here made `mergedList` usage of `<Pagination>` work.
I don't quite understand the mixing of `Props` and `propTypes`.

Reply:
`propTypes` is a React construct, and we are keeping it for components that have not been converted to TS yet. What error were you getting if you left this in?